python/setup.py

```diff
@@ -29,28 +40,130 @@
   'win32',
 ]
 
-cwd = os.path.abspath(os.path.dirname(__file__))
-with open(cwd + '/README.rst') as f:
-  readme = f.read()
-
-with open(cwd + '/requirements.txt') as f:
-  INSTALL_REQUIRES = [i.strip() for i in f.readlines()]
-
-with open(cwd + '/test_requirements.txt') as f:
-  TEST_REQUIRES = [i.strip() for i in f.readlines()]
-
-setup(name='sbp',
-      description='Python bindings for Swift Binary Protocol',
-      long_description=readme,
-      use_scm_version = {'root': '..', 'relative_to': __file__},
-      setup_requires=['setuptools_scm'],
-      author='Swift Navigation',
-      author_email='dev@swiftnav.com',
-      url='https://github.com/swift-nav/libsbp',
-      classifiers=CLASSIFIERS,
-      packages=PACKAGES,
-      platforms=PLATFORMS,
-      install_requires=INSTALL_REQUIRES,
-      tests_require=TEST_REQUIRES,
-      use_2to3=False,
-      zip_safe=False)
+MAJOR = 2
+MINOR = 4
+MICRO = 8
```

Comment:
Will these strings need to be updated by hand each release ^^ ?

Reply:
looks like the answer is yes

Reply:
I've fixed so that python version will be automatically generated along with the other languages (that already have version info automatically generated).